packages/frontend/src/page/index.tsx

```diff
@@ -6,7 +6,7 @@ import { Button, ErrorFeedback, Typography } from "@carrot-kpi/ui";
 import { ReactElement, useEffect, useState } from "react";
 import { OraclePage, KPITokenRemotePageProps } from "@carrot-kpi/react";
 import { CollateralData } from "../creation-form/types";
-import { Address, useNetwork, useProvider, useToken } from "wagmi";
+import { Address, useNetwork, usePublicClient, useToken } from "wagmi";
```

Comment:
We can import `Address` as a type.

Reply:
Addressed in 324e4eeffa1b055aabcfbd649a938a64786a3c3b